src/constants/chains.ts

```diff
@@ -27,7 +27,8 @@ export enum ChainsId {
   BTT = 199,
   AURORA = 1313161554,
   MOBM = 1284,
-  WAGMI = 11111
+  WAGMI = 11111,
+  COSTON = 16
 }
 
 export const CHAINS = {
```

Comment:
as per my last comment, lets remove this as its not required now

Reply:
Updated by @jb1011 ✅